src/Views/Error/Forbidden.cshtml

```diff
@@ -0,0 +1,13 @@
+ @{ ViewBag.Title = "Forbidden"; }
+
+<main role="main" class="govuk-main-wrapper" id="main-content">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">You are not permitted to see this page</h1>
+      <p class="govuk-body">This page is restricted to certain users only.</p>
+      <p class="govuk-body">
+        <a href="mailto:becomingateacher@digital.education.gov.uk?subject=Publish%20teacher%20training%20courses%20access%20request">Contact us by email to request access to your organisation</a>
```

Comment:
I'm pretty sure that link shouldn't be on the search side as it is in this diff

reject